### PR TITLE
Fix default options ApplicationParameters::class.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️

### CODE REVIEW

File: src/ApplicationParameters.php [hunk 1]
```diff
@@ -6,23 +6,23 @@
 
 final class ApplicationParameters
 {
-    private string $brandUrl;
-    private string $charset;
+    private string $brandUrl = '';
+    private string $charset = '';
```
Comment (samdark) [inline src/ApplicationParameters.php:10]: `utf-8` is better default value.

File: src/ApplicationParameters.php [hunk 2]
```diff
@@ -6,23 +6,23 @@
 
 final class ApplicationParameters
 {
-    private string $brandUrl;
-    private string $charset;
+    private string $brandUrl = '';
```
Comment (samdark) [inline src/ApplicationParameters.php:9]: `/` is better default value.

File: src/ApplicationParameters.php [hunk 3]
```diff
@@ -6,23 +6,23 @@
...
-    private string $heroFooterColumnLeft;
+    private string $heroFooterColumnLeft = '';
     private array $heroFooterColumnLeftOptions = [];
-    private string $heroFooterColumnRight;
+    private string $heroFooterColumnRight = '';
     private array $heroFooterColumnRightOptions = [];
-    private string $language;
-    private string $logo;
-    private string $name;
+    private string $language = '';
```
Comment (samdark) [inline src/ApplicationParameters.php:23]: `en` is better default value.

File: src/ApplicationParameters.php [hunk 4]
```diff
@@ -6,23 +6,23 @@
...
+    private string $logo = '';
+    private string $name = '';
```
Comment (samdark) [inline src/ApplicationParameters.php:25]: `My Application` is better default value.

File: src/ApplicationParameters.php (hunk 1 above)
Comment (terabytesoftw) [reply src/ApplicationParameters.php:10]: done.

File: src/ApplicationParameters.php (hunk 2 above)
Comment (terabytesoftw) [reply src/ApplicationParameters.php:9]: done.

File: src/ApplicationParameters.php (hunk 3 above)
Comment (terabytesoftw) [reply src/ApplicationParameters.php:23]: done.

File: src/ApplicationParameters.php (hunk 4 above)
Comment (terabytesoftw) [reply src/ApplicationParameters.php:25]: done.

**Pull request merged**